bouncer/blacklist/tests.py

```diff
@@ -0,0 +1,61 @@
+from django.test import TestCase
+from blacklist import models
+from blacklist.brain import is_ip_blacklisted, is_email_blacklisted
+
+
+class BrainTests(TestCase):
```

Comment:
Here, you can use the following method to create all the needed models (add some sample data) and then just add the checks in the tests.

You can do something like:

```
    @classmethod
    def setUpClass(cls):
        self.blacklisted_ip = "1.2.3.4"
        self.blacklisted_email = ...
        self.blacklisted_host = ...
        ... Create models ...
```